app/task_routes.py

```diff
@@ -0,0 +1,149 @@
+from requests import auth
+from app import db 
+from app.models.task import Task
+from flask import Blueprint, json, jsonify, make_response, request, abort
+import datetime, requests
+import os
+
+
+
+task_bp = Blueprint("task", __name__, url_prefix="/tasks")
+
+#Helper Functions 
+def valid_int(number, parameter_type):
```

Comment:
Note that this function is in `task_routes` and `goal_routes`. Consider including this function in a separate file such as `utils.py` and then you can import where its needed with `from app.utils import valid_int`